HW2/P5/driver.py

```diff
@@ -80,3 +103,4 @@ def randcolor():
             # SUBPROBLEM 3: sort objects by location.  Be sure to update the
             # grid if objects' indices change!  Also be sure to sort the
             # velocities with their object positions!
+            # at first, we create a
```

Comment:
Fair enough, totally understandable :)
Just a quick tip, surround the code you pasted above with "`" symbols so that it doesn't look absurd with GitHub's markdown and is readable